unit_tests/testVTKFiles/testVTKFiles.f90

```diff
@@ -360,6 +360,9 @@ PROGRAM testVTKFiles
          ALL((testVTKMesh%z .APPROXEQ. zref))
   ASSERT(bool, 'VTKMeshType OPERATOR(+)')
   WRITE(*,*) '  Passed: VTKMeshType OPERATOR(+)'
```

Comment:
Can we get rid of these WRITE statements throughout this test?  No reason for them since it's using the test harness now.